src/Symfony/Component/Message/HandlerLocator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message;
+
+use Symfony\Component\Message\Exception\NoHandlerForMessageException;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class HandlerLocator implements HandlerLocatorInterface
```

Comment:
to be consistent with previous suggestion, this would be a `MessageHandlerLocator`